setup.py

```diff
@@ -21,7 +21,7 @@
         'Framework :: Django :: 1.10',
         'Framework :: Django :: 1.11',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7'
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.7'
```

Comment:
If this still supports Python 2.7 and other versions of Python 3, then those could be left listed here as well.